frontend/demo/src/actions/conversations/index.ts

```diff
@@ -34,17 +37,10 @@ export const removeErrorFromConversationAction = createAction(
   resolve => (conversationId: string) => resolve({conversationId})
 );
 
-export interface FetchConversationsResponse {
-  data: ConversationPayload[];
-  metadata: ResponseMetadata;
-}
-
 export function fetchConversations() {
   return async (dispatch: Dispatch<any>) => {
     dispatch(loadingConversationsAction());
-    return doFetchFromBackend('conversations.list', {
-      page_size: 10,
-    })
+    return AiryHttpClient.fetchConversations()
```

Comment:
we should expose a payload that reflects https://docs.airy.co/api/http#list-conversations . To be fair the only way to know the exact payload at the moment is probably looking at the backend code. The point stays though: it's a library so we should expose, for example, the page size (maybe have a default sure) .